sdk/burdock/query/sql/private/query.py

```diff
@@ -19,7 +19,7 @@ def __init__(self, reader, metadata, epsilon=4.0):
         self.tau = 5
 
         #self.mechanism = Gaussian(epsilon, 100000, 1, self.tau)
-        self.mechanism = Laplace(self.epsilon, self.tau)
+        self.mechanism = Laplace(self.epsilon, tau=self.tau)
```

Comment:
I think we can just delete this and construct the mechanism where needed below, since the sensitivity is tied to each instance now.

Reply:
Deleted